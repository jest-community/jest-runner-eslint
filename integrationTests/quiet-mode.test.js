const runJest = require('./runJest');

it('does not log warnings as errors in quiet mode', async () => {
  expect(await runJest('quiet-mode')).toMatchSnapshot();
});
