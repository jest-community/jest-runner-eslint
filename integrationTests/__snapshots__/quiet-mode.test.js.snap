// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not log warnings as errors in quiet mode 1`] = `
"FAIL __eslint__/file.js
  ✕ no-undef 


/mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/quiet-mode/__eslint__/file.js
  5:3  error  'hello' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;
